src/connectors/connectStateResults.js

```diff
@@ -12,19 +18,9 @@ const connectStateResults = (renderFn, unmountFn) => (widgetParams = {}) => ({
   },
 
   render({ results, instantSearchInstance, state }) {
-    const resultsCopy = Object.keys(results).reduce((acc, key) => {
-      if (key !== '_state') {
-        // eslint-disable-next-line no-param-reassign
-        acc[key] = results[key];
-      }
-      return acc;
-    }, {});
+    const resultsCopy = _objectSpread({}, results);
```

Comment:
You do not remove the `_state` key from the copy anymore, is it normal?

Reply:
Yes, too much discussion on it, and the chance that someone will abuse it is small I think